recipes/watchdog-wms/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "watchdog-wms" %}
-{% set version = "1.2.4b" %}
+{% set version = "1.2.0" %}
```

Comment:
You are decreasing the version number here. This will not be the default version if people install watchdog.

Reply:
That's what I thought...I will change the version number. ;)